pennylane/ops/functions/iterative_qpe.py

```diff
@@ -15,30 +15,22 @@
 This module contains the qml.iterative_qpe function.
 """
 
-from warnings import warn
 
 import numpy as np
 
 import pennylane as qml
 
 
-def iterative_qpe(base, aux_wire="unset", iters="unset", ancilla="unset"):
+def iterative_qpe(base, aux_wire="unset", iters="unset"):
```

Comment:
These are meant to be required, positional arguments, and were converted into "required" keyword arguments as part of the deprecation.

```suggestion
def iterative_qpe(base, aux_wire, iters):
```